src/main/haskell/kore/src/Kore/AST/Common.hs

```diff
@@ -622,6 +630,14 @@ instance (Eq (Sort level), Eq (v level)) => Eq1 (Exists level v) where
         && (existsVariable a == existsVariable b)
         && liftedEq (existsChild a) (existsChild b)
 
+instance (Ord (Sort level), Ord (v level)) => Ord1 (Exists level v) where
+    liftCompare liftedCompare a b =
+        case compare (existsSort a) (existsSort b) of
```

Comment:
For this `Ord1` instance and the one below, you can use the `Monoid` instance for `Ordering`, which takes the first non-`EQ` result:
```.hs
   compare (existsSort a) (existsSort b)
<> compare (existsVariable a) (existsVariable b)
<> liftedCompare (existsChild a) (existsChild b)
```